server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/ConfigurationService.java

```diff
@@ -192,6 +192,8 @@
      */
     ConfigurationSchemaDto saveConfSchema(ConfigurationSchemaDto configurationSchema);
 
+    String validateConfiguration(String appId, int schemaVersion, String configurationBody);
```

Comment:
This method should be renamed to validateOverrideConfigurationBody ( validates configuration body by override schema). Also, this should return boolean or void (throw exception in case of validation error) if possible.